multi.go

```diff
@@ -115,17 +115,29 @@ func (m *MultiFinder) Collector(name string) (Collector, error) {
 	return Collector{}, nil
 }
 
-func (m *MultiFinder) Find(query Query) ([]File, error) {
-	// TODO: only send the query to the appropriate finders
-	files := []File{}
-	for proj, f := range m.getFinders() {
-		pF, err := f.Find(query)
-		if err != nil {
-			return nil, fmt.Errorf("find failed for %s: %v", proj, err)
-		}
-		files = append(files, pF...)
+func (m *MultiFinder) Find(query Query) ([]BGPDump, error) {
+	dumps := []BGPDump{}
+
+	// Extract the project name from the query's collectors
+	if len(query.Collectors) == 0 {
+		return nil, fmt.Errorf("no collectors specified in query")
+	}
+	projectName := query.Collectors[0].Project.Name
```

Comment:
Does this make project required now? It seems that the previous code supported queries across multiple projects?

Reply:
You are right -- `project` should default to all projects if not specified, instead of being a required field. 

The previous code would always query all projects even if we specify a particular project, which isn't what we want either. I will push a new fix.